subprojects/docs/src/docs/design/gradle-module-metadata-1.0-specification.md

```diff
@@ -85,6 +85,15 @@ This value must contain an array of 0 or more capabilities. Each capability is a
 #### Standard attributes
 
 - `org.gradle.usage` indicates the purpose of the variant. See the `org.gradle.api.attributes.Usage` class for more details. Value must be a string.
+- `org.gradle.status` indicates the kind of release: one of `release` or `integration`.
+
+#### Java Ecosystem specific attributes
+
+- `org.gradle.dependency.bundling` indicates how dependencies of the variant are bundled. Either externally, embedded or shadowed. See the `org.gradle.api.attributes.java.Bundling` for more details. Value must be a string.
```

Comment:
Is this Java ecosystem specific? the concept isn't. I can have a javascript library that embeds all of its dependencies in a script, or some of them, or none of them. Same for a native application. We just happened to start with JVM libraries.

If we do want this attribute to be JVM specific we should perhaps put something like 'jvm' in the name.

Reply:
Indeed, decided to move it to a top level attribute, see #8567